packages/next-templates/src/app/productpagina-klachtenformulier/page.tsx

```diff
@@ -0,0 +1,149 @@
+'use client';
+
+import {
+  Article,
+  BreadcrumbNav,
+  BreadcrumbNavLink,
+  ButtonLink,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { IconChevronRight, IconChevronUp } from '@tabler/icons-react';
+
+import '@utrecht/design-tokens/dist/index.css';
+import '@/app/styling/css/productpagina-klachtenformulier.css';
+import '@/app/styling/css/wmebv.css';
+import '@/app/styling/css/detail-page.css';
+
+import { useEffect } from 'react';
+
+export default function home() {
+  const pageTitle = 'Klacht over de gemeente doorgeven';
+  const websiteLabel = 'gemeente Voorbeeld';
+
+  useEffect(() => {
+    if (typeof document?.title === 'string') {
+      document.title = `${pageTitle} - ${websiteLabel}`;
+    }
+  }, []);
+
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+
+      <PageContent className="voorbeeld-page-content-flex">
+        <Article id="main" className="voorbeeld-article-space ">
+          <BreadcrumbNav label="Kruimelpad">
+            <BreadcrumbNavLink href="/wmebv#">Terug</BreadcrumbNavLink>
+          </BreadcrumbNav>
+          <Heading1>{pageTitle}</Heading1>
+          <Paragraph lead>
```

Comment:
Is er geen design token die dit text dikker kan maken? Volgens mij maakt 'lead' de paragraaf groter of dikker

Reply:
Dit heb ik overgenomen van de contactpagina van wmebv omdat ik in het Figma ontwerp zag staan dat het paragraph lead is:
<img width="1033" alt="Scherm­afbeelding 2024-03-21 om 14 53 10" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/ca42ab54-a446-4a55-8017-cc8064c4ff54">

Is dit niet iets wat op elke pagina zo is (zou moeten zijn?)
→ ik heb even rondgekeken op de website en in de code en ook dit is wederom niet consistent. Op sommige pagina's is de eerste paragraph "lead" en op andere pagina's niet. Op de pagina's waar het gebruikt wordt, is het wel op dezelfde manier gedaan als hoe ik het heb gedaan. 

In Storybook zie ik bij de css componenten wel iets over lead en tokens staan (https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-paragraph--docs#lead). Ik heb in de code base gezocht naar utrecht-paragraph--lead, en kon deze niet vinden. Bij react componenten zie ik juist lead staan zoals ik het gebruikt heb (https://nl-design-system.github.io/utrecht/storybook/?path=/docs/react_react-paragraph--docs#lead-paragraph). Omdat dit op meerdere plekken op deze manier toegepast wordt denk ik dat dit de juiste manier is. Maar not sure, ik zal er een issue van aanmaken.